Makefile

```diff
@@ -0,0 +1,69 @@
+.PHONY: all clean tarball srpm rpm
+
+PACKAGE_NAME = perl-WWW-Gooddata
+
+VERSION ?= $(shell git describe --tags --match '*.*.*' | tr -- '-' '_')
```

Comment:
I want to have same (release) version in each Perl module. After release of some version there will be a git tag with version.